freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/model/SynchronizeAllUsersRequest.java

```diff
@@ -18,6 +18,9 @@
     @ApiModelProperty(value = UserModelDescriptions.USERSYNC_USER_CRNS)
     private Set<String> users = new HashSet<>();
 
+    @ApiModelProperty(value = UserModelDescriptions.DELETED_WORKLOAD_USER)
+    private String deletedWorkloadUser;
```

Comment:
is there any reason this can't be a set? it looks more flexible and I don't see why can't we delete 2 user at once

Reply:
It has been resolved but not answered. I'm pushing this because a set would be more flexible and later you can't change this as the API has to be backward compatible.

Reply:
Sorry, my comment got lost. I did originally consider a set here, but ended up going with a single user (and had discussed with David prior to posting the PR) since it aligns with some follow on work coming up. We plan on updating user sync to only allow environment level sync or single actor sync and dropping support for something in between (like syncing N arbitrary users). This would involve adding new api endpoints and probably deprecating these existing ones. (I can point you to the design doc if you want). So I think it makes sense to keep this field single user only.

Reply:
Okay, after further discussion, I've updated to set and added a validation that will fail if size isn't 1.